services/order.py

```diff
@@ -0,0 +1,30 @@
+from django.db import transaction
+from db.models import Order, Ticket, User
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None) -> Order:
```

Comment:
The `date` parameter is currently typed as `str`. Consider using `datetime` for better handling of date and time values, which will also align with the `DateTimeField` used in the `Order` model.